usagov/usagov-2021
# USAGOV-1411 combo box

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1441

## Description

Added JS to hide the clear button automatically added by uswds for the combo-box in contact elected officials. Removed the hard coded clear button for the state governments page.  
Removed unused html-blocks for contact elected officials.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- On /elected-officials check that the combo-box for states does not have a clear button once a selection is made. 
- On /state-governments check that the combo-box for state does not have a clear button once a selection is made.

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
